spec/features/book_index_redirect_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Book index redirect', type: :feature do
+  before(:each) do
+    create(:book)
+    Book.reindex
+
+    visit root_path
+  end
+
+  context 'user search:' do
+    it 'specific world' do
+      fill_in 'Book title', with: 'Yard'
+      click_button 'Search'
+      expect(page).to have_current_path('/searches?search=Yard')
+    end
+
+    it 'nothing' do
+      fill_in 'Book title', with: ''
```

Comment:
and don't repeat yourself. For example, you can use `let(:title) { book.title }` and then you can move part of your test to the before block:
```
before do
  fill_in 'Book title', with: title
  click_button 'Search'
end
```